core/src/test/scala/unit/kafka/coordinator/group/GroupCoordinatorTest.scala

```diff
@@ -480,11 +480,11 @@ class GroupCoordinatorTest extends JUnitSuite {
 
   @Test
   def staticMemberRejoinWithLeaderIdAndKnownMemberId() {
-    val rebalanceResult = staticMembersJoinAndRebalance(leaderInstanceId, followerInstanceId)
+    val rebalanceResult = staticMembersJoinAndRebalance(leaderInstanceId, followerInstanceId, sessionTimeout = DefaultRebalanceTimeout / 2)
```

Comment:
Not clear why we want to override to half of the value?

Reply:
We are just trying to make sure we timeout the follower instance because default session timeout = default rebalance timeout